README.md

```diff
@@ -70,6 +71,7 @@ Knowledge graphs connect data points into a rich semantic web of entities and re
 
 - [DBpedia](https://wiki.dbpedia.org/) ![Status](https://img.shields.io/badge/status-active-brightgreen) ![Language](https://img.shields.io/badge/language-PHP-blueviolet) - A project aimed at extracting structured content from Wikipedia.
 - [Google Knowledge Graph](https://developers.google.com/knowledge-graph) ![Status](https://img.shields.io/badge/status-active-brightgreen) - Powering Google's search engine with structured knowledge representation.
+- [Louie.AI, AI KG](https://www.louie.ai/) ![Status](https://img.shields.io/badge/status-active-brightgreen) ![Language](https://img.shields.io/badge/language-Python-blue) ![Language](https://img.shields.io/badge/CUDA-76B900?logo=nvidia&logoColor=white) Real-time self-adjusting AI graph intelligence tier for popular scalable vector-capable databases with native support for scalable ingestion, targeting, analysis, and triggering.
```

Comment:
Hi @lmeyerov , 

I don't think louie.ai is an example of a knowledge graph. The goal here is to have a knowledge graph that users can query.

Reply:
Hi denironyx, part of the backend we are piloting is a real-time AI KG , think self-adjusting graph RAG with logical & semantic relations. We perform various data extraction & integration steps like entity extraction, resolution , geolocation, etc, and hooks for doing various triggering and compute on it. 

We do not do sparql/rdf/etc, but we do storage of entities, relations, and more modern semantic layers over that. Sort of how Facebook had to build its graph on top of SQL & friends instead of a graph db, we are building our KG on top of KV stores & vector stores.

Hope that helps!

Reply:
Thanks for the clarity! This make sense.